opencraft/settings.py

```diff
@@ -579,7 +579,7 @@
 
 if 'file' in HANDLERS:
     LOGGING['handlers']['file'] = {
-        'level': 'DEBUG',
+        'level': 'INFO',
```

Comment:
This is the only log level we care about.  I'd prefer if we only change this one, and maybe the Huey log level, since the Huey debug log messages are really not that useful.  (They actually were quite useful when I had to debug a Huey bug, but we haven't run into any in a while.)

Reply:
Oh yep, sorry, didn't think to just change the file handler since that's what Filebeat will interact with. Will leave the Huey log level at `INFO` then too.